gitapp/lib/main.dart

```diff
@@ -10,20 +12,14 @@ class MyApp extends StatelessWidget {
   @override
   Widget build(BuildContext context) {
     return MaterialApp(
-      title: 'Flutter Demo',
-      theme: ThemeData(
-        // This is the theme of your application.
-        //
-        // Try running your application with "flutter run". You'll see the
-        // application has a blue toolbar. Then, without quitting the app, try
-        // changing the primarySwatch below to Colors.green and then invoke
-        // "hot reload" (press "r" in the console where you ran "flutter run",
-        // or simply save your changes to "hot reload" in a Flutter IDE).
-        // Notice that the counter didn't reset back to zero; the application
-        // is not restarted.
-        primarySwatch: Colors.blue,
+      title: 'Gitapp',
+      theme: ThemeData.dark().copyWith(
+        primaryColor: Color(0xFF2D239F),
+        scaffoldBackgroundColor: Color(0xFF313EBB),
+
+        brightness: Brightness.dark
       ),
-      home: MyHomePage(title: 'Flutter Demo Home Page'),
+      home: MyHomePage(title: 'Gitapp'),
```

Comment:
please extract `MyHomePage` to another file. Also we should not use anything like `My`. My means whose?
It is not mine, yours. It is app state so it should be called simple `HomePage`

Reply:
solved